src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -9,55 +9,85 @@ folder: che-6/user-guide
+To generate the SSH keys, take the following steps:
 
-*IMPORTANT!*. Hostname is an actual hostname of a VCS provider. Examples: `github.com`, `bitbucket.org`.
+. Click *Profile* > *Preferences* > *SSH* > *VCS*.
 
-*Use Existing SSH Keys*
+. Click *Generate Key* and manually save the resulting key to your Git hosting provider account. 
 
-You can upload an existing public key instead of creating a new SSH key. When uploading a key add the hostname (using no *www or http/https* - as in the example below). Note that the `public key > view` button will not be available with this option as the public file should be generated already.
+. When prompted to provide the hostname for your repository, ensure that it is a bare hostname (without www or http/https). Hostname is an actual hostname of a VCS provider. Example: `github.com`, `bitbucket.org`
+
+After the key is generated, you can view and copy it, and save it to your repository hosting account.
+
+[id='using-existing-ssh-keys']
+=== Using existing SSH keys
+
+You can upload an existing public key instead of creating a new SSH key. When uploading a key, add the hostname (without www or http/https). Note that the *View* button in the *Public Key* column will not be available with this option because the public file should be generated already.
 
 image::git/ssh_keys.gif[]
 
-The following example is specific to GitHub and GitLab but can be used with all git or SVN repository providers that use SSH authentication. Please refer to documentation provided by other providers for additional assistance.
+The following example is specific to GitHub and GitLab, but it can be used with any Git or SVN repository providers that use SSH authentication. For additional information, refer to the documentation provided by other providers.
+
+*GitHub example*
+
+If you do not want to set up GitHub oAuth, you can manually upload a key. Add the associated public key to a repository or account using *github.com*. 
+
+. Click your user icon in the top-right corner. 
+
+. Click *Settings*  > *SSH and GPG keys* > *New SSH key*. 
 
-*GitHub Example*
+. In the *Title* field, type a title for the key. 
 
-If you don’t want to setup GitHub oAuth, you can manually upload a key. Add the associated public key to a repository/account using *github.com*, click your user icon(top right) then `settings > ssh and gpg keys > new ssh key`. Give a title to your liking and paste the public key copied from Che into form.
+. In the *Key* field, paste the public key copied from Che. 
+
+. Click *Add SSH Key*.
 
 image::git/ssh_github.png[]
 
-*GitLab Example*
+*GitLab example*
+
+To add the associated public key to a git repository or account using *gitlab.com*, take the following steps:
```

Comment:
"git repository" -> "Git repository"

`gitlab.com` is marked up with bold, previously marked up with monospace.